index.html

```diff
@@ -395,5 +395,40 @@ <h6>Skills</h6>
         </div>
       </section>
     </main>
+    <footer>
+      <div class="container-about-me">
+        <form action="https://formspree.io/f/xoqbwkba" method="post">
+          <ul class="form-list">
+            <li><h2>Contact me</h2></li>
+            <li>
+              <p>
+                If you have an application you are interested in developing, a
+                feature that you need built or a project that needs coding. I’d
+                love to help with it
+              </p>
+            </li>
+            <li class="form-list-text">
+              <input type="text" id="username" name="username" placeholder="username" maxlength="30" />
+            </li>
+            <li class="form-list-text">
+              <input type="email" id="email" placeholder="email" name="email" />
```

Comment:
- [ ] Kindly use the placeholder corresponding to your chosen design template 👍🏽 .

Figma Design 👇🏽 
 
![image](https://user-images.githubusercontent.com/63473584/193897733-5939cacc-03cb-40a4-83c8-7830053acdbb.png)

Your Design 👇🏽 

![image](https://user-images.githubusercontent.com/63473584/193897902-6fac7040-cd70-480b-95d6-5956d24a8780.png)